### PR TITLE
Suggest text cleanup

### PR DESCRIPTION
**Pull request by ehsteve**

### CODE REVIEW

File: SEP-0009.md [hunk 1]
```diff
@@ -3,71 +3,59 @@
...
-
-The desire for stability and predictability is countered by the desire of the
-SunPy developers to continue providing new features and functionality and
-continue to iterate on the SunPy API to further increase our usability and
-correctness.
-
-This SEP therefore tries to focus on when the SunPy core package will be
-released rather than placing any strict limitations on changes and API
-stability.
+The focus of this SEP is not to prescribe how quickly the core package through
```
Comment (Cadair) [inline SEP-0009.md]: This dosen't read right to me?

File: SEP-0009.md [hunk 2]
```diff
@@ -3,71 +3,59 @@
...
+for 6 months or until the next release. The second release of the year
+will be an LTS release and will be supported for 12 months or until
+the next LTS release. Support periods can be extended beyond the requirements
+here if needed.
 
 The objective of this support cycle is to balance the desire to provide a
 release cycle for people who wish to experience minimal changes to SunPy with
-minimizing the maintenance burden on the SunPy developers. It is feasible that
-we could extend the support cycle of the LTS release to longer than 12 months,
-to give users an overlap period where they can upgrade. However, at this time,
```
Comment (Cadair) [inline SEP-0009.md]: I *really* want to keep the "However, at this time, supporting more than two releases simultaneously is beyond the resources of the project." I think that's it is important we make it plainly clear that we can't support this kind of thing given our current (lack of) funding structure.

Comment (ehsteve) [reply SEP-0009.md]: But I really don't think that belongs in an SEP...how about we say something like this release schedule is consistent with the current available resources and should be revisited if more resources become available?

File: SEP-0009.md (hunk 1 above)
Comment (ehsteve) [reply SEP-0009.md]: Fixed

File: SEP-0009.md (hunk 2 above)
Comment (ehsteve) [reply SEP-0009.md]: Done.

**Pull request merged**